features/git-town/configuration/configuration_automatic.feature

```diff
@@ -27,13 +27,31 @@ Feature: Automatically running the configuration wizard if Git Town is unconfigu
     | git sync-fork      |
 
 
+  Scenario Outline: Proceeding to configuration upon initial config prompt (using prompt's default)
```

Comment:
I don't understand the title for this scenario outline. Do you mean "Chosing the default option for the choice whether to run the set up wizard"?